dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSFServerTest.java

```diff
@@ -114,6 +99,13 @@ public void testLibertyWebConfigProviderFactory() throws Exception {
     @Test
     @SkipForRepeat("JSF-2.3")
     public void testLibertyWebConfigProvider() throws Exception {
+        WebClient webClient = new WebClient();
+        URL url = JSFUtils.createHttpUrl(jsfTestServer1, contextRoot, "");
+        // Ensure the isErrorPagePresent message is logged in the trace during the RESTORE_VIEW phase.
+        HtmlPage page = (HtmlPage) webClient.getPage(url);
+        //Basic check to confirm the app is working
+        assertTrue(page.asText().contains("Hello World"));
+
         String msgToSearchFor = "isErrorPagePresent ENTRY";
 
         // Check the trace.log to see if the LibertyWebConfigProvider has any entry trace.
```

Comment:
Can you give some explanation as to why we need to do this? Why did the test fail when that request was driven after we search for the message in the logs? Does the request trigger the message in the logs?

Reply:
Talked with Vlad, basically the test requires a request to be driven and usually we're lucky and the testServlet test runs first however order is not defined for  test execution so this change makes sense.